### ISSUE TITLE
UI Scaling - Broken button images

### ISSUE THREAD

**Issue by enigmafyv** [Anuken/Mindustry#2554]
**Platform**: *Android/iOS/Mac/Windows/Linux*
Android (v8) - LG V20

**Build**: *The build number under the title in the main menu. Required.*
104.10

**Issue**: *Explain your issue in detail.*
Scale UI to 150% or larger causes image artifacts to appear instead of original icon/image.  (see attached)

**Steps to reproduce**: *How you happened across the issue, and what exactly you did to make the bug happen.*
Settings > Graphics > UI Scaling > set scale to 150% or larger

**Link(s) to mod(s) used**: *The mod repositories or zip files that are related to the issue, if applicable.*
n/a

**Save file**: *The save file you were playing on when the bug happened, if applicable.*
n/a

**Crash report**: *The contents of relevant crash report files. REQUIRED if you are reporting a crash.*
n/a

---

*Place an X (no spaces) between the brackets to confirm that you have read the line below.*  
- [X] **I have searched the closed and open issues to make sure that this problem has not already been reported.**

---
***-- Attachments --***
**Scale 150%**
![Mindustry_Scale_150_percent](https://user-images.githubusercontent.com/4110691/93109330-8faa4d80-f679-11ea-88e0-344ceb9a23b8.png)

**Scale 175%**
![Mindustry_Scale_175_percent](https://user-images.githubusercontent.com/4110691/93109332-9042e400-f679-11ea-8a1d-7c53c1d9b3f8.png)

**Comment by Gdeft** [comment]
You ran out of the texture space thing, also why are you doing that anyways.

**Comment by Anuken** [comment]
Duplicate of #2047

**Comment by enigmafyv** [comment]
Buttons are too small on my phone, so I logically thought, hey make the UI bigger and then I can easily hit the button.  I found that even making the UI slightly bigger still was problematic because when I would hit where the bigger button is visually located, I'd still hit a button to its right.  It's only if I pressed the larger button slightly to its image's left, then I would get the correct response.
